ppr-ui/src/composables/mhrInformation/useMhrInformation.ts

```diff
@@ -71,6 +71,8 @@ export const useMhrInformation = () => {
 
     getMhrTransferHomeOwnerGroups.value.forEach(ownerGroup => {
       if (ownerGroup.action !== ActionTypes.REMOVED || isDraft) {
+        ownerGroup.interestDenominator = ownerGroup.interestDenominator || 0
```

Comment:
Won't this affect the field validation? When they are null, the validation kicks in, but if they are 0's what happens?

Reply:
This only occurs on saving after validation.  The API was returning an error if these values were null..